Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -539,17 +567,17 @@
 #elif ENABLED(SWITCHING_TOOLHEAD)
   #define E_STEPPERS      EXTRUDERS
   #define E_MANUAL        EXTRUDERS
-#elif ENABLED(PRUSA_MMU2)
+#elif HAS_PRUSA_MMU2
   #define E_STEPPERS 1
 #endif
 
-// No inactive extruders with MK2_MULTIPLEXER or SWITCHING_NOZZLE
-#if EITHER(MK2_MULTIPLEXER, SWITCHING_NOZZLE)
+// No inactive extruders with SWITCHING_NOZZLE or Průša MMU1
+#if ENABLED(SWITCHING_NOZZLE) || HAS_PRUSA_MMU1
```

Comment:
@thinkyhead I'm wondering what is better between `#if ENABLED(SWITCHING_NOZZLE) || HAS_PRUSA_MMU1` and `#if EITHER(SWITCHING_NOZZLE, HAS_PRUSA_MMU1)`. Both seems to compile correctly but HAS_PRUSA_MMU1 is not a directly activated option (and there is a mix in code that I don't like, I would prefer a uniform approach)

Reply:
The function of these `ENABLED` (etc.) macros is to digest any config switches and any switch-like conditionals, converting them into ones, zeros, and boolean operators. These switch-like conditionals can be used naked in pre-processor directives, as you would expect, but if you want to use them in code, for example a simple `if (HAS_PRUSA_MMU1)` then you must wrap it one of the `ENABLED` macros. It's a matter of stylistic choice whether to also apply them in the pre-compiler directives where they are not required.